includes/admin/class-wcmp-api.php

```diff
@@ -159,11 +160,23 @@ public function getShipmentLabels(array $shipment_ids, array $order_ids, array $
 
         if ($display) {
             $collection->setPdfOfLabels($positions);
+
             WCMP_Export::saveTrackTracesToOrders($collection, $order_ids);
+            foreach ($order_ids as $order_id) {
+                $order = WC_Core::get_order($order_id);
+                (new WCMP_Export())->getShipmentData($collection->getConsignmentIds(), $order);
+            }
+
             $collection->downloadPdfOfLabels($display);
         } else {
             $collection->setLinkOfLabels($positions);
+
             WCMP_Export::saveTrackTracesToOrders($collection, $order_ids);
```

Comment:
ik denk dat het hier misgaat
```suggestion
            WCMP_Export::saveTrackTracesToOrders($collection, $order_ids);
```
```suggestion
            WCMP_Export::saveTrackTracesToOrders($collection, $order_ids);
```